baseui/templates/baseui/head.html

```diff
@@ -0,0 +1,32 @@
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="icon" type="image/png" href="{{ url_for('baseui.static', filename='images/icons/favicon-32x32.png')}}" sizes="32x32" />
+<link rel="icon" type="image/png" href="{{ url_for('baseui.static', filename='images/icons/favicon-16x16.png')}}" sizes="16x16" />
+<title>{{ pagetitle }}</title>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
+<link rel="stylesheet" href="{{ url_for('baseui.static', filename='css/arxivstyle.css')}}" />
+<script type="text/x-mathjax-config">
```

Comment:
I could see breaking out the mathjax tags into a separate template fragment. But not essential.

Reply:
Good point. Kind of hard to draw the line, at this moment, on which pieces are big enough for their own fragment. Perhaps when we're closer to done we can review for sensible pieces?

Reply:
Makes sense to me!